content/posts/web-components.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Portable Spectrograms and Verification Interface"
+date: 2025-03-27T22:00:00+10:00
+image: "/images/ecosounds-banner-image.jpg"
+imageCredit: "QUT Ecoacoustics Staff"
+tags:
+    - News
+    - Technical
+---
+
+Open Ecoacoustics have recently published portable web components that can be
+embedded into any website.
+The web components provide Ecoacoustic tools such as spectrograms, annotation
+viewers, and customizable verification grids.
```

Comment:
Let do a short description of what a web component is. Very non-technical description here.

- What are web components?
- Are a standard way to package up bit of a website and allow them to be reused on other websites. We call each package a component
- Why?
- implemented spectrogram, annotation rendering several times before. 
- each time was custom, required work, waste of time
- now, open source, hope everyone can take advantage of them
- still early days, contributions welcome, bugs exist

Reply:
below the fold